kubernetes/helm/ckeditor-pdf-converter/README.md

```diff
@@ -0,0 +1,48 @@
+# CKEditor PDF Converter On-Premises Helm chart
+
+Use this Helm chart to provision CKEditor PDF Converter on your
+Kubernetes cluster.
+
+## Minimum requirements
+- 1 CPU Core
+- 512MB RAM
+- Kubernetes 1.19+
+- Helm v3
+
+The default configuration of running this service requires the reservation of 1
+CPU core and 512MB of RAM in the cluster. For more information about resources
+usage look here:
```

Comment:
```suggestion
The default configuration of running this service requires the reservation of 2
CPU cores and 1024MB of RAM in the cluster. For more information about resources
usage look here:
```

Reply:
My review changed this part of the content, so these will need to be reintroduced by hand.

Reply:
Fixed in commit.